src/audio/audio.cpp

```diff
@@ -486,16 +498,30 @@ void Audio::doCapture()
     }
 #endif
 
-    for (size_t i = 0; i < AUDIO_FRAME_SAMPLE_COUNT * AUDIO_CHANNELS; ++i)
-        buf[i] *= inGain;
+    for (quint32 i = 0; i < AUDIO_FRAME_SAMPLE_COUNT * AUDIO_CHANNELS; ++i)
+    {
+        int16_t ampPCM = static_cast<int16_t>(qRound(buf[i] * gain));
```

Comment:
Given that the gain can be as high as 20 now, this line does not stop the samples from railing against `INT16_MAX` (or it's negative), insteading allowing it to roll over, creating horrible distortion instead of digital peaking.